ActindoWorkFlow/Integration/01_ConfigureQueueTypes.md

```diff
@@ -0,0 +1,373 @@
+
+  ![Create queue type](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/CreateQueueType.png "[Create queue type]")
+
+2. Enter a name for the queue type in the *Name* field.
+
+3. Enter the desired number of workers for the queue type in the *Number of workers* field.
+
+  > [Info] A worker is a job executing the actions within a process. One worker can only execute one action at the same time. By assigning multiple workers to a queue type, this queue type may execute several actions at the same time. Consequently, actions with a queue type with more workers are executed faster that actions with queue types with fewer workers. The total number of workers of all queue types depends on the number of booked vCores.   
+  For one worker, 4 vCores are needed. Additionally, 10 % of the booked vCores, but at least 4 vCores are blocked for the daily business and cannot be used for workers. The number of vCores needed always refers to the total number of workers in all queue types. For example, for a queue type with 2 workers and a queue type of 4 workers, 28 vCores are needed:   
+  *4 vCores per worker (4 * 6 = 24) plus 4 vCores blocked for the daily business (24 + 4 = 28)*
+
+
+4. Click the *Execution Priority* drop-down list and select the appropriate execution priority. The following options are available:
+  - **Oldest executable action**: The oldest executable actions within all existing processes is executed first by the workers.
+  - **Executable actions of oldest process**: The executable actions within the oldest process are executed first by the workers.
+
+
+5. Click the [SAVE] button in the upper right corner.   
+  The *Create Queue Type* view is closed. The new queue type is created and displayed in the list of queue types. The *Queue type has been created* pop-up window is displayed.
+
+  ![Queue type created](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/QueueTypeCreated.png "[Queue type created]")
+
+  > [Info] An error notice is displayed if the number of booked vCores is not sufficient to create the queue type with the selected number of workers.
+
+  ![Maximum total workers](/Assets/Screenshots/ActindoWorkFlow/QueueTypes/MaximumTotalWorkers.png "[Maximum total workers]")
+
+### Next steps
+
+- [Disable the workflows](#disable-the-workflows)
+- [Edit a queue type](#edit-a-queue-type)
+- [Assign a queue type](#assign-a-queue-type)
+- [Delete a queue type](#delete-a-queue-type)
+- [Kill workers](#kill-workers)
+- [Configure the logging setting](01_ConfigureLoggingSettings.md)
+
+### See also
+
+- [User Interface workflows](/ActindoWorkFlow/UserInterface/00_UserInterface.md)
+- [Workflow and process elements](/ActindoWorkFlow/Overview/02_WorkflowProcessElements.md)
+- [Manage the workflows](/ActindoWorkFlow/Operation/01_ManageWorkflows.md)
+
+
+
+
+## Disable the workflows
+
+Disable the workflows so that no new workflow processes are started which may be necessary to edit the queue types or workers.
```

Comment:
```suggestion
Disable the workflows so that no new workflow processes are started, which may be necessary to edit the queue types or workers.
```
Ich würde Komma da schreiben, da erklärender Nebesatz.